packages/xod-client/src/editor/containers/Patch/modeUtils.js

```diff
@@ -3,7 +3,14 @@ import * as R from 'ramda';
 export const bindApi = (api, fn) => (...args) => fn(api, ...args);
 
 // :: Point -> String
-export const getOffsetMatrix = ({ x, y }) => `matrix(1, 0, 0, 1, ${x}, ${y})`;
+export const getOffsetMatrix = ({ x, y }) =>
+  `matrix(1, 0, 0, 1, ${Math.round(x)}, ${Math.round(y)})`;
+
+// :: Point -> Point
+export const roundOffsetToPixel = R.compose(
```

Comment:
It’s a function operating over `Point`’s (not limited to offsets). Is `roundPointToPixel` a better name? Just a bore.